### PR TITLE
traefik: remove requirement storage class

### PR DESCRIPTION
**Pull request by hectorj2f**
We don't need storage for traefik

### CODE REVIEW

File: addons/traefik/1.7.x/traefik-1.yaml [hunk 1]
```diff
@@ -6,9 +6,6 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: traefik
     kubeaddons.mesosphere.io/provides: ingresscontroller
-    # TODO: we're temporarily supporting dependency on an existing default storage class
-    # on the cluster, this hack will trigger re-queue on Addons until one exists.
-    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
     catalog.kubeaddons.mesosphere.io/addon-revision: "1.7.2-1"
```
Comment (jieyu) [inline addons/traefik/1.7.x/traefik-1.yaml]: BTW, can you also fixing the version here. We're using traefik 1.7.12. We can do that in a separate PR.

**Comment by hectorj2f** [comment]
I had a discussion with Sam I will block this PR and until a further discussion happens I will install a storage addon even if I don't need it in my case 😂

**Comment by hectorj2f** [comment]
In relation to this discussion, Jimmi proposed to disable traefik acme and use cert manager acme instead.

**Comment by hectorj2f** [comment]
I am closing this PR, we'll apply the changes to the new releases rather than backport it to beta-8.
